pkg/sbom/v1/sbom_spdx_storage_format.go

```diff
@@ -128,14 +139,38 @@ func (sbom *SBOMData) StoreFilteredSBOMName(name string) {
 	sbom.filteredSpdxData.ObjectMeta.SetName(name)
 }
 
-func (sbom *SBOMData) StoreMetadata(instanceID instanceidhandler.IInstanceID) {
-	labels := instanceID.GetLabels()
+func (sbom *SBOMData) StoreMetadata(wlidData string, imageID string, instanceID instanceidhandler.IInstanceID) {
+	labels := map[string]string{
+		namespaceLabelKey:     wlid.GetNamespaceFromWlid(wlidData),
+		kindLabelKey:          wlid.GetKindFromWlid(wlidData),
+		nameLabelKey:          wlid.GetNameFromWlid(wlidData),
+		containerNameLabelKey: instanceID.GetContainerName(),
+	}
 	for i := range labels {
 		if labels[i] == "" {
 			delete(labels, i)
+		} else {
+			errs := validation.IsValidLabelValue(labels[i])
+			if len(errs) != 0 {
+				logger.L().Debug("label is not valid", helpers.String("label", labels[i]))
+				for j := range errs {
+					logger.L().Debug("label err description", helpers.String("Err: ", errs[j]))
+				}
+				delete(labels, i)
+			}
 		}
 	}
 	sbom.filteredSpdxData.ObjectMeta.SetLabels(labels)
+
+	annotations := make(map[string]string)
+	if imageID != "" {
+		annotations[imageIDKeyForAnnotations] = imageID
+	}
+	instanceIDString := instanceID.GetStringFormatted()
+	if instanceIDString != "" {
+		annotations[instanceIDKeyForAnnotations] = instanceIDString
```

Comment:
issue: label value is inconsistent with Kubevuln

According to the constants definition:
```go
const (
	labelPrefix                 = "kubescape.io"
	Organization                = "Organization"
	Tool                        = "Tool"
	Person                      = "Person"
	KubescapeOrganizationName   = "Kubescape"
	KubescapeNodeAgentName      = "KubescapeNodeAgent"
	RelationshipContainType     = "CONTAINS"
	namespaceLabelKey           = "/workload-namespace"
	kindLabelKey                = "/workload-kind"
	nameLabelKey                = "/workload-name"
	containerNameLabelKey       = "/workload-conatienr-name"
	imageIDKeyForAnnotations    = "imageID"
	instanceIDKeyForAnnotations = "instanceID"
)
```

So `instanceIDKeyForAnnotations` is `instanceID`. However, this is inconsistent with keys that [Kubevuln uses](https://github.com/kubescape/kubevuln/blob/e6f88e14e4a894503915473cb5e056581d8dc6a7/core/domain/sbom.go#L9) to store such annotations and values that we agreed with David.

The Instance ID annotation should be:
```
"kubescape.io/instance-id": "instanceID-value"
```

Reply:
@matthyx it is ok by you that I'll add the instance id annotations key to the instanceidhandler package under the k8s-interface repo, so we will both take it from there?

Reply:
ok